composer.json

```diff
@@ -255,6 +260,16 @@
             "type": "vcs",
             "url": "https://github.com/Gizra/og.git"
         },
+        {
+            "_comment": "This directly references the repository. This can be removed once a release is made and the repository is announced on Packagist.",
+            "type": "vcs",
+            "url": "https://github.com/openeuropa/oe_webtools_location.git"
```

Comment:
I see it's registered by Packagist: https://packagist.org/packages/openeuropa/oe_webtools_location